lib/link.js

```diff
@@ -10,7 +10,22 @@ const link = (directories, cwd = process.cwd()) => {
 
     fs.exists(dist, (exists) => {
       if (exists) { return; }
-      fs.symlinkSync(src, dist, 'dir');
+
+      fs.stat(cwdSrc, (err, stats) => {
+        if (err && err.code === 'ENOENT') {
+          return;
+        }
+
+        if (stats.isFile()) {
+          fs.symlinkSync(src, dist, 'file');
+        }
+
+        if (stats.isDirectory()) {
```

Comment:
I would like to keep a default case.
In case of an error the developer should be informed directly by throwing it. Without the default case the code will neither do anything nor does it inform the developer that something went wrong. Debugging will be really hard in such a case.